Project 1/aleksandrajablonska

aleksandra-jablonska opened the pull request:
Coś mi nie pykło niestety do końca, proszę o podpowiedź, gdzie jest błąd (obstawiam html z divami, bo odkleja mi się pomarańczowy box od dwóch pozostałych).

aleksandra-jablonska committed:
index.html modified

aleksandra-jablonska committed:
index.html modified

aleksandra-jablonska committed:
style.css added

aleksandra-jablonska committed:
google fonts added

aleksandra-jablonska committed:
index.html modified

aleksandra-jablonska committed:
index.html and style.css modified

aleksandra-jablonska committed:
index.html and style.css modified

aleksandra-jablonska committed:
index.html and style.css modified

aleksandra-jablonska committed:
style.css modified

ToJurek commented:
Hej, widzę, że jest nie ma problemu z parańczowym cardem. U mnie to sie trochę bardziej rozjechało, także podeślij mi screena jak u Cb wygląda.
<img width="776" alt="image" src="https://user-images.githubusercontent.com/50409767/224478478-9a986386-c93c-45c4-9dcf-44c38f53da32.png">

Jeżeli chodzi o całość, to poprawnie zostały zdefiniowane selekory i przypisane do elementów. Poza tym widać, że skupiasz sie na stylach głebok zagnieźdzonych i warto byłoby jeszcze trochę je pozmieniac w ramach nauki i dostosowac paddingi tak, aby np. bloki tekstu zaczynały się w tym samym miejscu. Ogólnie dobra robota, daj znać jak to u Cb wygląda

ToJurek commented on Project1/preview-cards/index.html:
```diff
@@ -1,32 +1,124 @@
+        <div class="tiles-row">
+          <div class="orange">
+            <div class="icon">
+              <svg width="64" height="40" xmlns="http://www.w3.org/2000/svg">
```
to jest jeden ze sposobów użycia svg, bedzię to szczególnie uzyteczne w projektach gdzie dbamy o responsywność ikon. Także plus dla Ciebie, ale pamietaj, że do importu obrazu możemy używac też elementu `img`. 🙂

aleksandra-jablonska replied:
ok, zapamiętam! :)

ToJurek commented on Project1/preview-cards/index.html:
```diff
@@ -1,32 +1,124 @@
+                </g>
+              </svg>
+            </div>
+            <div class="title">
```
bardzo dobre nazwy klas. Są jednoznaczne oraz idealnie odzwierciedlaja zawartośc 🚀

ToJurek commented on Project1/preview-cards/index.html:
```diff
@@ -1,32 +1,124 @@
+              </svg>
+            </div>
+            <div class="title">
+              <h2>LUXURY</h2>
```
Implementacja poprawna, a pytanie do Cb czy można za pomocą CSS zrobić tak, aby `LUXURY` był w pełeni zapisany małymi literami? Jeśli tak, to jak to zrobić?

aleksandra-jablonska replied:
text-transform: lowercase;

ToJurek replied:
Dokładnie o to mi chodziło 👍 podrzucam link do dokumentacji jakby ktos dowiedzi PRke https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

ToJurek commented on Project1/preview-cards/style.css:
```diff
@@ -0,0 +1,64 @@
+    font-size: 15px
+    line-height: 25px;
+    font-family: "Big Shoulders Display", cursive;
+    font-family: "Lexend Deca", sans-serif;
```
są zdefiniowane dla font-family. Musi niestety zostać jeden

aleksandra-jablonska commented:
Cześć,

w załączniku przesyłam Ci zrzucik – u mnie tak to wyglądało. Zastanawia
mnie, dlaczego w momencie chęci wyśrodkowania całego elementu (żeby całość
znalazła się na środku strony), pomarańczowy element odchodził od reszty
(przesuwały się tylko te dwa „niebieskie”.

Ja nie jestem zadowolona z tego zadania 😜 wiem, że nawaliłam i czuję się
jak przegryw. Najśmieszniejsze jest to, że w grudniu stworzyłam tę swoją
stronkę, która, mimo że nie jest idealna, znacznie wykracza poziomem
zaawansowania od tego zadania. Natomiast przyznaję się, że od grudnia nie
miałam już styczności ani z HTML, ani z CSS, więc podejrzewam, że „wyszłam
z wprawy”. Wczorajsze zajęcia sporo mi przypomniały, dlatego przez weekend
wykonam to zadanie jeszcze raz 🙂

PS. Podeślę Ci w mailu kod do mojej strony (justme.com.pl). Ostrzegam, że
pisałam całość w notatniku 😜 przy tworzeniu posiłkowałam się oczywiście
informacjami z kursu, który wtedy przerobiłam i skrypty z javy to gotowce
znalezione w internecie.


sob., 11 mar 2023 o 11:14 Tomek ***@***.***> napisał(a):

> Hej, widzę, że jest nie ma problemu z parańczowym cardem. U mnie to sie
> trochę bardziej rozjechało, także podeślij mi screena jak u Cb wygląda.
> [image: image]
> <https://user-images.githubusercontent.com/50409767/224478478-9a986386-c93c-45c4-9dcf-44c38f53da32.png>
>
> Jeżeli chodzi o całość, to poprawnie zostały zdefiniowane selekory i
> przypisane do elementów. Poza tym widać, że skupiasz sie na stylach głebok
> zagnieźdzonych i warto byłoby jeszcze trochę je pozmieniac w ramach nauki i
> dostosowac paddingi tak, aby np. bloki tekstu zaczynały się w tym samym
> miejscu. Ogólnie dobra robota, daj znać jak to u Cb wygląda
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/infoshareacademy/jfddr9-projekt-html-css/pull/3#issuecomment-1464878182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A556KXJ2PWLH22A36I5RVWLW3RGBFANCNFSM6AAAAAAVU6AW4Y>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

*{
   margin: 0;
    padding: 0;
    box-sizing: border-box;
}

img {
    max-width: 100%;
    max-height: 100%;
    vertical-align: middle;
}

.main {
    font-family: Poppins, sans-serif; 
    color: rgb(18, 36, 50);
    background: rgb(230, 213, 145);
}

a {
    color: rgb(204, 167, 82);
    text-decoration: none;
}

h1, h2 {
    font-weight: 700;
    font-size: 40px;
    line-height: 60px;
    margin-bottom: 14px;
}

@media (min-width: 800px) {
    h1, h2 {
        font-size: 52px;
        line-height: 78px;
        margin-bottom: 20px;
    }
}

h3 {
    font-size: 24px;
    line-height: 36px;
    margin-bottom: 8px;
}

p {
    font-size: 15px;
    line-height: 22px;
}

.label {
    color: rgba(18, 36, 50, 0.48);
    font-size: 16px;
    line-height: 28px;
    font-weight: 600;
    margin-bottom: 4px;
}

@media (min-width: 800px) {
    .label {
        margin-bottom: 10px; 
    }
}

.big-text {
    margin-bottom: 34px;
}

@media (min-width: 800px) {
    .big-text {
        font-size: 24px;
        line-height: 38px;
        margin-bottom: 54px; 
    }
}

.button {
    display: inline-block;
    text-align: center;
    font-size: 15px;
    line-height: 20px;
    font-weight: 700;
    padding: 20px 25px;
    border-radius: 30px;
}

.button-dark {
    background: rgb(204, 167, 82);
    color: rgb(64, 64, 64);
}

.button-light {
    color:rgb(204, 167, 82);
    border: 2px solid rgb(18, 36, 50);
}

.button-cta {
    color: rgb(255, 255, 255);
    background: rgb(204, 167, 82);
    padding: 12px 25px;
    border-radius: 22px;
}

.container {
    max-width: 1220px;
    margin: 0 auto;
    padding: 0 15px;
}

.row {
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
    margin-left: -15px;
    margin-right: -15px;
}

@media (min-width: 800px) {
    .row {
        flex-direction: row;
    }
}

.col {
    width: 100%;
    padding-left: 15px;
    padding-right: 15px;
}

@media (min-width: 800px) {
    .col-1-2 {
        flex: 0 0 50%;
    }

    .col-1-3 {
        flex: 0 0 33.3333%;
    }
}

@media (min-width: 800px) {
    .col-reversed {
        order: -1;
    }
}

.text-center {
    text-align: center;
}

.text-right {
    text-align: right;
}

.relative {
    position: relative;
}

.pt-std {
    padding-top: 43px;
}

@media (min-width: 800px) {
    .pt-std {
        padding-top: 71px;
    }
}

.pb-std {
    padding-bottom: 43px;
}

@media (min-width: 800px) {
    .pb-std {
        padding-bottom: 71px;
    }
}

.pt-sm {
    padding-top: 32px;
}

@media (min-width: 800px) {
    .pt-sm {
        padding-top: 71px;
    }
}

.pb-sm {
    padding-bottom: 32px;
}

@media (min-width: 800px) {
    .pb-sm {
        padding-bottom: 71px;
    }
}

.navigation {
    position: relative;
    height: 78px;
    background: rgb(255, 255, 255);
    border-radius: 39px;
    display: flex;
    align-items: center;
    padding: 0 25px;
    margin-top: 30px;
    box-shadow: 0px 4px 32px rgba(0, 16, 61, 0.08);
    z-index: 2;
}

@media (min-width: 972px) {
    .navigation {
        height: 114px;
        border-radius: 57px;
        margin-top: 50px;
        padding: 0 40px;
    }
}

@media (min-width: 972px) {
    .navigation-logo {
        margin-right: 60px;
    }
}

.menu {
    display: none;
}

@media (min-width: 972px) {
    .menu {
        display: flex;
        align-items: center;
    }
}

.menu-button {
    display: none;
}

@media (min-width: 972px) {
    .menu-button {
        margin-left: auto;
        display: block;
    }
}

.menu a {
    margin-right: 67px;
    font-weight: 600;
}

.mobile-menu {
    display: none;
    position: absolute;
    left: 15px;
    right: 15px;
    top: 69px;
    background: rgb(255, 255, 255);
    height: 87vh;
    z-index: 1;
    border-radius: 0 0 39px 39px;
}

.mobile-menu a {
    margin-bottom: 53px;
    font-weight: 600;
}

.mobile-menu a:last-child {
    margin-bottom: 0;
}

.mobile-menu-visible {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

#mobile-menu-toggler {
    margin-left: auto;
    cursor: pointer;
}

@media (min-width: 972px) {
    #mobile-menu-toggler {
        display: none;
    }
}

#mobile-menu-wrapper {
    visibility: hidden;
    position: fixed;
    top: 0;
    bottom: 0;
    right: 0;
    left: 0;
    background: rgba(0, 16, 61, 0.84);
}

.mobile-menu-wrapper-visible {
    visibility: initial !important;
}

.site-top {
    background: rgb(230, 213, 145);
}

.intro {
    padding: 43px 0;
}

@media (min-width: 800px) {
    .intro {
        padding: 110px 0;
    }
    .intro-left {
        padding-top: 60px;
    }
}

.intro-right {
    padding-top: 55px;
}

@media (min-width: 800px) {
    .intro-right {
        padding-top: 0;
    }
}

.feature-img {
    margin-bottom: 32px;
}
/* cookie alert */
#cookies-message{
	color: white; padding: 12px 10px; text-align: center; position: fixed; bottom:0px; left:0; right:0; background-color: rgba(0,0,0,0.6); z-index: 100000; box-shadow: 0 0 5px rgba(0,0,0,0.4); display: none;
}
#accept-cookies-checkbox{
	background-color: #00AFBF; color: #FFF; border: solid 1px #00AFBF; transition: all 0.5s; padding: 2px 6px; border-radius: 4px; display: inline-block; margin-left: 10px; text-decoration: none; cursor: pointer
}
#accept-cookies-checkbox:hover{
	background-color: transparent; border-color: white;
}

footer {
    padding-top: 60px;
}

@media (min-width: 800px) {
    footer {
        padding-top: 100px;
    }
}

.footer-row {
    padding: 55px 0 100px;
}

@media (min-width: 800px) {
    .footer-row {
        padding: 60px 0 117px;
    }
}

.footer-left {
    margin-bottom: 25px;
    font-size: 14px;
}

.footer-left address {
    font-style: normal;
}

@media (min-width: 800px) {
    .footer-left {
        margin-bottom: 0;
    }
}

.footer-right {
    display: flex;
    align-items: flex-start;
    justify-content: flex-end;
}

@media (min-width: 800px) {
    .footer-right {
        text-align: right;
    }
}

.footer-right p {
    font-size: 12px;
}

.hr:before {
    content: "";
    display: block;
    width: 100%;
    height: 3px;
    background: rgb(204, 167, 82);
    margin: 0 auto;
}
<!DOCTYPEhtml>
<html lang="pl">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="style.css">
    <title>Just Me – Usługi copywriterskie</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link ***@***.***;600;700&amp;display=swap" rel="stylesheet"> 
</head>
<body class="main">
    <div class="site-top">
        <div class="container">
            <div class="row">
                <div class="col relative">
                    <div class="navigation">
                        <img class="navigation-logo" src="Just Me.svg" alt="Just Me – Usługi Copywriterskie">
                        <nav class="menu">
                            <a href="index.html"><strong>Strona główna</strong></a>
                            <a href="Blog.html">Blog</a>
                            <a href="Oferta.html">Oferta</a>
                            <a href="Kontakt.html">Kontakt</a>
                        </nav>
                        <div class="menu-button">
                            <a class="button button-cta" href="Kontakt.html">Skontaktuj się ze mną!</a>
                        </div>
                        <span id="mobile-menu-toggler" onclick="navToggle();">
                            <img src="hamburger.png" alt="">
                        </span>
                    </div>
                    <div id="mobile-menu-wrapper"></div>
                    <div id="mobile-menu" class="mobile-menu text-center">
                        <a href="index.html"><strong>Strona główna</strong></a>
                        <a href="Blog.html">Blog</a>
                        <a href="Oferta.html">Oferta</a> 
                        <a href="Kontakt.html">Kontakt</a>
                        <a class="button button-cta" href="Kontakt.html">Skontaktuj się ze mną!</a>
                    </div>
                </div>
            </div>
        </div>
 <section class="intro">
            <div class="container">
                <div class="row">
                    <div class="col col-1-2 intro-left">
                        <span class="label">Pozwól sobie pomóc</span>
                        <h1>Chcesz współpracować z najlepszymi?</h1>
                        <p class="big-text">Napisz do mnie już dziś: ***@***.***</p>
                        <p class="big-tex">Preferujesz kontakt telefoniczny? Zadzwoń do mnie i rozpocznijmy udaną współpracę: 534 870 434!
                        <a class="button button-dark" href="Kontakt.html">Skontaktuj się ze mną!</a>
                    </div>
                    <div class="col col-1-2 intro-right text-right">
                        <img src="JustMelogo.png" alt="Just Me">
                    </div>
                </div>
            </div>
        </section> 
    <footer>
        <div class="container hr">
            <div class="row footer-row">
                <div class="col col-1-2 footer-left">
                    <img src="JustMelogo.png" alt="">
                    <address>ul. Kaliska 44/27, 56-500 Syców</address>
                </div>
                <div class="col col-1-2 footer-right">
                    <p>© Just Me Aleksandra Jabłońska, 2023. All rights reserved.<br> Obrazki zastosowane w projekcie są użyte jedynie w celach wizualizacyjnych.</p>        
                </div> 
            </div>
        </div>
    </footer>
<script>
    function navToggle() {
        var menu = document.getElementById('mobile-menu');
        var wrapper = document.getElementById('mobile-menu-wrapper')
        menu.classList.toggle('mobile-menu-visible');
        wrapper.classList.toggle('mobile-menu-wrapper-visible');
    }
</script>
</body>
</html>
<!DOCTYPEhtml>
<html lang="pl">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="style.css">
    <title>Just Me – Usługi copywriterskie</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="Twojej firmie potrzebny profesjonalny copywriter? Koniecznie zapoznaj się z moją ofertą!">
    <meta name="keywords" content="copywriter, marketing, SEO, pozycjonować">
    <meta name="author" content="Just Me Aleksandra Jabłońska">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link ***@***.***;600;700&amp;display=swap" rel="stylesheet">
</head>
<body class="main">
    <div class="site-top">
        <div class="container">
            <div class="row">
                <div class="col relative">
                    <div class="navigation">
                        <img class="navigation-logo" src="naw.png" alt="Just Me – Usługi copywriterskie">
                        <nav class="menu">
                            <a href="index.html"><strong>Strona główna</strong></a>
                            <a href="Cennik.html">Cennik</a>
                            <a href="Oferta.html">Oferta</a>
                            <a href="Kontakt.html">Kontakt</a>
                        </nav>
                        <div class="menu-button">
                            <a class="button button-cta" href="Kontakt.html">Skontaktuj się ze mną!</a>
                        </div>
                        <span id="mobile-menu-toggler" onclick="navToggle();">
                            <img src="hamburger.png" alt="">
                        </span>
                    </div>
                    <div id="mobile-menu-wrapper"></div>
                    <div id="mobile-menu" class="mobile-menu text-center">
                        <a href="index.html"><strong>Strona główna</strong></a>
                        <a href="Cennik.html">Cennik</a>
                        <a href="Oferta.html">Oferta</a> 
                        <a href="Kontakt.html">Kontakt</a>
                        <a class="button button-cta" href="Kontakt.html">Skontaktuj się ze mną!</a>
                    </div>
                </div>
            </div>
        </div>
        <section class="intro">
            <div class="container">
                <div class="row">
                    <div class="col col-1-2 intro-left">
                        <span class="label">Pozwól sobie pomóc</span>
                        <h1>I zainwestuj w treści, które sprzedają!</h1>
                        <p class="big-text">Napisz do mnie i otrzymaj najlepszy tekst!</p>
                        <a class="button button-light" href="Kontakt.html">Skontaktuj się ze mną!</a>
                    </div>
                    <div class="col col-1-2 intro-right text-right">
                        <img src="JustMelogo.png" alt="Just Me">
                    </div>
                </div>
            </div>
        </section> 
    </div>
    <section class="features">
        <div class="container">
            <div class="row">
                <div class="col text-center pt-std pb-std">
                    <span class="label">Wszystko, czego potrzebujesz</span>
                    <h2>Dlaczego Just Me?</h2>
                </div>
            </div>
            <div class="row pb-sm">
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-1.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..napiszę dla Ciebie treści, które doskonale sprzedadzą Twój produkt lub usługę.</p>
                </div>
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-2.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..skutecznie zareklamuję Twoją markę i trafię do Twoich potencjalnych klientów.</p>
                </div>
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-3.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..rzeczowo i profesjonalnie poprowadzę Twojego firmowego bloga.</p>
                </div>
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-4.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..stworzę ciekawe i angażujące treści na Twoje kanały w social mediach.</p>
                </div>
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-5.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..poprowadzę dla Ciebie kampanię reklamową na Facebooku</p>
                </div>
                <div class="col col-1-3 pb-std">
                    <img class="feature-img" src="feature-6.png" alt="">
                    <h3>Bo tylko ja..</h3>
                    <p>..stworzę dla Ciebie profesjonalną stronę internetową i zapełnię ją zoptymalizowaną treścią sprzedażową.</p>
                </div>
            </div>
        </div>
    </section>
    <section class="info-1">
        <div class="container">
            <div class="row">
                <div class="col col-1-2 pb-sm">
                    <span class="label">Jak zwiększyć rozpoznawalność swojej marki i zyskać klientów?</span>
                    <h2>Potrzebujesz angażujących treści</h2>
                    <p class="big-text">Teksty opisujące markę i ofertę powinny wykorzystywać język korzyści dostosowany do potrzeb grupy docelowej Twoich klientów. Takie działania pozwolą przekonać odbiorców, że potrzebują z współpracować właśnie z Twoją firmą!</p>
                    <a class="button button-light" href="Oferta.html">Dowiedz się więcej</a>
                </div>
                <div class="col col-1-2 pb-sm">
                    <img src="images/platnosci.png" alt="">
                </div>
            </div>
            <div class="row pt-std">
                <div class="col col-1-2 pb-sm">
                    <span class="label">Ty również współpracuj tylko z najlepszymi!</span>
                    <h2>Profesjonalny copywriter to połowa sukcesu Twojej firmy</h2>
                    <p class="big-text">Doskonała oferta bez odpowiednich działań marketingowych nie obroni się na konkurencyjnym rynku. Już dzisiaj wypłyń na powierzchnię i rozwiń swój biznes dzięki unikalnym, dobrze pozycjonującym treściom!</p>
                    <a class="button button-light" href="Oferta.html">Dowiedz się więcej</a>
                </div>
                <div class="col col-1-2 col-reversed">
                    <img src="images/karty.png" alt="">
                </div>
            </div>
        </div>
    </section>
    <footer>
        <div class="container polityki">
                <div class="row">
                <div class="col relative">
                <div class="navigation">
                            <a href="politykaprywatnosci.html"><strong>Polityka prywatności</strong></a>
</div>
</div>
</div>
</div>
        <div class="container hr">
                <div class="col col-1-2 footer-left">
                    <p>© Just Me Aleksandra Jabłońska, 2023. All rights reserved.<br></p> 
                    <p>Obrazki zastosowane w projekcie są użyte jedynie w celach wizualizacyjnych.</p>        
                </div> 
            </div>
        </div>
    </footer>
<script>
    function navToggle() {
        var menu = document.getElementById('mobile-menu');
        var wrapper = document.getElementById('mobile-menu-wrapper')
        menu.classList.toggle('mobile-menu-visible');
        wrapper.classList.toggle('mobile-menu-wrapper-visible');
    }
</script>
<div id="cookies-message">Ta strona używa ciasteczek (cookies), dzięki którym nasz serwis może działać lepiej oraz w sposób dostosowany do indywidualnych potrzeb. W każdej chwili możesz zmienić ustawienia dotyczące cookies w ustawieniach swojej przeglądarki.<a onclick="WHCloseCookiesWindow();" id="accept-cookies-checkbox">Rozumiem</a></div>

<script>
function WHCheckCookies(){
	if(!localStorage.cookies_accepted) {
		document.getElementById("cookies-message").style.display = "block"
	} 
}
function WHCloseCookiesWindow(){
	localStorage.cookies_accepted = true;
	document.getElementById("cookies-message").remove();
}
	
window.onload = WHCheckCookies;
</script>
</body>
</html>
